config/application.rb

```diff
@@ -22,7 +22,10 @@ class Application < Rails::Application
     config.time_zone = 'London'
 
     # Custom directories with classes and modules you want to be autoloadable.
-    config.autoload_paths += %W(#{config.root}/lib)
+    config.autoload_paths += %W(
+                                #{config.root}/lib
+                                #{config.root}/app/services/supergroups
```

Comment:
This is great because it follows the pattern in Collections, but our previous work on Services and News (I think) puts everything in lib for now. It may make sense to follow that pattern for now and perhaps create a card to move everything into `/services` later.